tuna/rocmlir/rocmlir_lib.py

```diff
@@ -122,6 +129,14 @@ def compose_worker_list(self, machines) -> Optional[List[RocMLIRWorker]]:
 
       #determine number of processes by compute capacity
       worker_ids: List = machine.get_avail_gpus()
+      factor = self.args.load_factor
+      if factor > 1:
+        # LF>1 means multiple workers per GPU.
+        worker_ids = worker_ids * round(factor)
+      else:
+        # LF<1 means use that fraction of GPUs.
+        worker_ids = worker_ids[:round(len(worker_ids) * factor)]
```

Comment:
Say there is only 1 GPU and LF is 0.5. Then worker_ids will become an empty list. Does that mean the GPU won't be used at all and the script would just quit? Is this the desired behavior?